scenario/scenario_shacl.ttl

```diff
@@ -230,28 +148,35 @@ scenario:StructuralShape a sh:NodeShape ;
             sh:order 0 ;
             sh:path scenario:catalogs ],
         [ skos:example "http://mywebsite.de/myModel.fbx" ;
+            sh:datatype xsd:anyURI ;
             sh:description "Link to 3D models"@en ;
             sh:message "Validation of 3dModels failed!"@en ;
             sh:minCount 0 ;
-            sh:name "3dModels"@en ;
+            sh:name "3d models"@en ;
             sh:node general:LinkShape ;
             sh:order 1 ;
             sh:path scenario:3dModels ],
         [ skos:example "http://mywebsite.de/map.xodr" ;
+            sh:datatype xsd:anyURI ;
             sh:description "Link to traffic space"@en ;
             sh:maxCount 1 ;
             sh:message "Validation of trafficSpace failed!"@en ;
             sh:minCount 0 ;
             sh:name "traffic space"@en ;
             sh:node general:LinkShape ;
             sh:order 2 ;
-            sh:path scenario:trafficSpace ] ;
-    sh:targetClass scenario:Structural .
-
-scenario:TrafficShape a sh:NodeShape ;
-    sh:closed false ;
-    sh:ignoredProperties ( rdf:type ) ;
-    sh:property [ skos:example "kangaroo crossing" ;
+            sh:path scenario:trafficSpace ],
+        [ skos:example "194.01" ;
+            sh:datatype xsd:float ;
+            sh:description "Azimuth of the sun in degrees"@en ;
+            sh:message "Validation of sunAzimuth failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "sun azimuth"@en ;
+            sh:order 2 ;
```

Comment:
In dieser Shape haben wir zweimal `sh:order 2`.
Gibt weitere Stellen, an denen die Order nicht passt, zwei weitere habe ich angemerkt. Am besten schau nochmal gesamthaft drüber.

Reply:
fixed